### DISCUSSION TITLE
Is there any way to disable smooth scroll animation when seeking?

### DISCUSSION THREAD

**Discussion by paweldepta** [katspaugh/wavesurfer.js#3808] [Q&A]
I have enabled auto scrolling and auto centering the waveform, but I need to disable the scroll animation when clicking on the waveform to seek forward.
```javascript
    autoScroll: true,
    autoCenter: true,
    hideScrollbar: true,
 ```
 
 Is this possible? If not, maybe there is any way of changing the duration of the animation?

**Comment by katspaugh** [comment]
You could try turning auto-centering off using setOptions on click or on pause and then turning it back on play.
